### PR TITLE
Homework3: Dmitry Kiselev 

### PR DESCRIPTION
**Pull request by KiselevDV**

### CODE REVIEW

File: HomeWork_3/Task_3.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+'''Найти самое длинное слово в введенном предложении.
+Учтите что в предложении есть знаки препинания.'''
+line = str(input('Enter string \n'))
+list_words = line.split()
+LongWord = 0
+for i in range(1, len(list_words)):
```
Comment (belovmd) [inline HomeWork_3/Task_3.py:11]: can't we use for i in list_words?

File: HomeWork_3/Task_3.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+'''Найти самое длинное слово в введенном предложении.
+Учтите что в предложении есть знаки препинания.'''
+line = str(input('Enter string \n'))
+list_words = line.split()
+LongWord = 0
```
Comment (belovmd) [inline HomeWork_3/Task_3.py:10]: this naming contradicts python codestyle we usually use snake case for vars

File: HomeWork_3/Task_3.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+'''Найти самое длинное слово в введенном предложении.
+Учтите что в предложении есть знаки препинания.'''
```
Comment (belovmd) [inline HomeWork_3/Task_3.py:4]: Your solution doesn't handle punctuation marks

File: HomeWork_3/Task_3.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+'''Вводится строка. Требуется удалить из нее
+повторяющиеся символы и все пробелы. Например,
+если было введено "abc cde def", то должно быть
+выведено "abcdef".'''
+text = str(input('Enter string \n'))
+text = text.strip()
+text = text.split()
+text = ''.join(text)
+print(text)
```
Comment (belovmd) [inline HomeWork_3/Task_3.py]: Your solution doesn't remove letters duplicates

**Review by belovmd — CHANGES_REQUESTED** [review]
1. remove redundant files from PR
2. Please address next issues

File: HomeWork_3/Task_3.py (hunk 1 above)
Comment (KiselevDV) [reply HomeWork_3/Task_3.py]: Почему?
Всё работает!

File: HomeWork_3/Task_3.py (hunk 3 above)
Comment (KiselevDV) [reply HomeWork_3/Task_3.py:4]: line = str(input('Enter string \n'))
import string
for c in string.punctuation:
    line = line.replace(c,'')
print(line)
list_words = line.split()
LongWord = 0
for i in range(1, len(list_words)):
    if len(list_words[LongWord]) < len(list_words[i]):
        LongWord = i
LongWord = list_words[LongWord].strip(",")
print(LongWord)

File: HomeWork_3/Task_3.py (hunk 4 above)
Comment (KiselevDV) [reply HomeWork_3/Task_3.py]: text = str(input('Enter string \n'))
text = text.split()
text = ''.join(text)
text_nondublicate = []
for i in text:
    if i not in text_nondublicate:
        text_nondublicate.append(i)
print(''.join(text_nondublicate))

**Pull request closed without merging**